activerecord/test/cases/null_relation_test.rb

```diff
@@ -38,7 +38,7 @@ def test_async_query_on_null_relation
   end
 
   def test_none_chained_to_methods_firing_queries_straight_to_db
-    assert_no_queries do
+    assert_no_queries(ignore_none: false) do
```

Comment:
This can now trigger some SCHEMA queries:

```ruby
Failure:
NullRelationTest#test_none_chained_to_methods_firing_queries_straight_to_db [/Users/byroot/src/github.com/Shopify/rails/activerecord/test/cases/null_relation_test.rb:41]:
2 instead of 0 queries were executed.
Queries:
PRAGMA table_info("developers")
SELECT sql FROM
  (SELECT * FROM sqlite_master UNION ALL
   SELECT * FROM sqlite_temp_master)
WHERE type = 'table' AND name = 'developers'
.
Expected: 0
  Actual: 2
```

Which I think is fine.